WPMedia/fusion-news-theme-blocks
# add translations for ar, bn, bo, id, km-kh, lo, ms, my, pt-br, th, ug…

…, vi, zh-cn, zh-tw languages

## Jira Ticket

- [THEMES-1412](https://arcpublishing.atlassian.net/browse/THEMES-1412)

## Acceptance Criteria

_copy from ticket_

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout themes-1412`
2. Change locale in blocks.json to a newly supported one like `ms` 
3. Add that same locale to the localeList in the blocks.json
4. Run fusion repo with linked blocks `npx fusion start -f -l` (needs at minimum output type block and one other)
5. Check to make sure that a block with translations available renders with the correctly translated phrases

## Effect Of Changes

## Dependencies or Side Effects

Feature pack PR: https://github.com/WPMedia/arc-themes-feature-pack/pull/495
manifest PR: https://github.com/WPMedia/arc-themes-manifests/pull/224

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [ ] Confirmed all the test steps a reviewer will follow above are working. -- my blocks are stale so I cannot conirm this
        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1412]: https://arcpublishing.atlassian.net/browse/THEMES-1412?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
